app/views/layouts/application.html.haml

```diff
@@ -33,3 +36,33 @@
     .container
       = render "layouts/shared/notification"
       = yield
+      -if current_user.present? && current_user.role.to_s == "Admin"
+        = form_for @company, remote: true do |f|
```

Comment:
Why this form exist in application layout?

Read this http://railsapps.github.io/rails-default-application-layout.html and http://guides.rubyonrails.org/layouts_and_rendering.html